src/main/java/com/vistatec/ocelot/di/OcelotModule.java

```diff
@@ -94,10 +104,13 @@ protected void configure() {
         bind(RuleConfiguration.class).toInstance(ruleConfig);
         bind(PluginManager.class).toInstance(pluginManager);
         bind(TmManager.class).toInstance(tmManager);
-        bind(TmPenalizer.class).to(SimpleTmPenalizer.class);
+        bind(TmPenalizer.class).toInstance(penalizer);
+        bind(TmService.class).toInstance(tmService);
+        bind(TmGuiManager.class).toInstance(tmGuiManager);
 
         bindServices(eventQueue, cfgService, docStats);
```

Comment:
Minor thing, not about Marta's changes, more about Kevin's work -- I don't understand why we split some bind calls off into `bindServices()` and do others directly.